### PR TITLE
Update messages

### PR DESCRIPTION
**Pull request by tk0miya**
I updated error messages commented on transifex.
I need someone's review because my English skill is very poor :(

### CODE REVIEW

File: sphinx/application.py [hunk 1]
```diff
@@ -205,9 +205,9 @@ def __init__(self, srcdir, confdir, outdir, doctreedir, buildername,
                 self.config.setup(self)
             else:
                 raise ConfigError(
-                    _("'setup' that is specified in the conf.py has not been "
-                      "callable. Please provide a callable `setup` function "
-                      "in order to behave as a sphinx extension conf.py itself.")
+                    _("'setup' that is defined in the conf.py, is not a Python callable. "
+                      "Please provide a callable `setup` function in order to make conf.py "
+                      "behave as a Sphinx extension.")
```
Comment (tk0miya) [inline sphinx/application.py]: Another proposal:
"'setup' defined in the conf.py is not a function. Sphinx expects it is a function. Please check your settings."

Comment (jfbu) [reply sphinx/application.py]: proposal

    _("'setup'" as currently defined in conf.py isn't a Python callable. "
      "Please modify its defintion to make it a callable function. This is "
      "needed for conf.py to behave as a Sphinx extension.")

File: sphinx/application.py [hunk 2]
```diff
@@ -515,7 +515,7 @@ def add_event(self, name):
 
     def set_translator(self, name, translator_class):
         # type: (unicode, Type[nodes.NodeVisitor]) -> None
-        logger.info(bold(_('A Translator for the %s builder is changed.') % name))
+        logger.info(bold(_('A translator for the %s builder is changed.') % name))
```
Comment (jfbu) [inline sphinx/application.py]: proposal

    _('Change of translator for the %s builder.') % name

File: sphinx/application.py (hunk 1 above)
Comment (jfbu) [reply sphinx/application.py]: oops there was a typo in my proposal

    _("'setup'" as currently defined in conf.py isn't a Python callable. "
      "Please modify its definition to make it a callable function. This is "
      "needed for conf.py to behave as a Sphinx extension.")

**Comment by tk0miya** [comment]
Updated and merged.
Thank you for comment!

**Pull request merged**